### PR TITLE
feat(StatusSeedPhraseView): Update layout

### LINKED ISSUE #5669: [Onboarding] Update SeedPhraseInput Layout
### Description

Update onboarding SeedPhraseInput component according the the new designs:
https://www.figma.com/proto/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=4160%3A160332&scaling=min-zoom&page-id=2499%3A150211&starting-point-node-id=4160%3A160332

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #5669

Depends on: https://github.com/status-im/StatusQ/pull/663

### What does the PR do
Updates SeedInputPhrase layout due to new designs

### Affected areas
Onboarding/SeedInputPhrase

### Screenshot of functionality
https://user-images.githubusercontent.com/31625338/167887814-f41754a9-cbdb-45d6-930e-273b077ff356.mov

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Code looks good and everything still works as far as my tests go.

File: ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml [hunk 1]
```diff
@@ -188,10 +192,9 @@ OnboardingBasePage {
 
             delegate: StatusSeedPhraseInput {
                 id: seedWordInput
-                width: (grid.cellWidth - 24)
-                height: (grid.cellHeight - 28)
-                textEdit.input.anchors.leftMargin: 16
-                textEdit.input.anchors.rightMargin: 16
+                width: (grid.cellWidth - 8)
+                height: (grid.cellHeight - 8)
+                Behavior on width { NumberAnimation { duration: 180 } }
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml:197]: This seems a little distracting/off putting. Is the transition necessary?

**Review by PascalPrecht — APPROVED** [review]
Cool stuff!

**Comment by Hbouaz** [comment]
Looks good:

image

**Review by Hbouaz — APPROVED** [review]

**Pull request merged**